101-datastore-create-blob/azuredeploy.json

```diff
@@ -0,0 +1,110 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"workspaceName": {
+			"type": "string",
+			"metadata": {
+				"description": "Specifies the name of the Azure Machine Learning workspace which will hold this datastore target."
+			}
+		},
+		"datastoreName": {
+			"type": "string",
+			"metadata": {
+				"description": "The name of the datastore, case insensitive, can only contain alphanumeric characters and underscore"
+			}
+		},
+		"accountName": {
+			"type": "string",
+			"metadata": {
+				"description": "The name of the storage account."
+			}
+		},
+		"containerName": {
+			"type": "string",
+			"metadata": {
+				"description": "The name of the azure blob container."
+			}
+		},
+		"storageAccountSubscriptionId": {
+			"type": "string",
+			"defaultValue":"",
+			"metadata": {
+				"description": "Optional : The subscription ID of the storage account."
+			}
+		},
+		"storageAccountResourceGroup": {
+			"type": "string",
+			"defaultValue":"",
+			"metadata": {
+				"description": "Optional : The resource group of the storage account."
+			}
+		},
+		"accountKey": {
+			"type": "securestring",
+			"defaultValue":"",
+			"metadata": {
+				"description": "Optional : A storage account key, defaults to None. Skip this if SAS token authentication is used."
+			}
+		},
+		"sasToken": {
+			"type": "securestring",
+			"defaultValue":"",
+			"metadata": {
+				"description": "Optional : An account SAS token, defaults to None. Skip this if storage account key authentication is used."
+			}
+		},
+		"endPoint": {
+			"type": "string",
+			"defaultValue":"",
+			"metadata": {
+				"description": "Optional : The endpoint of the storage account. Defaults to core.windows.net"
+			}
+		},
+		"protocol": {
+			"type": "string",
+			"defaultValue":"",
+			"metadata": {
+				"description": "Optional : Protocol to use to connect to the blob container. If not specified, defaults to https."
+			}
+		},
+		"skipValidation": {
+			"type": "bool",
+			"defaultValue":false,
+			"allowedValues": [
+				false,
+				true
+			],
+			"metadata": {
+				"description": "Optional : If set to true, the call will skip Datastore validation. Defaults to false"
```

Comment:
what does validation do?  (and why are we skipping it in CI?)

Reply:
Flag skips connection settings & data validation in ADLS store, ADLS Gen2 file-syste,  file share, SQL, PostgreSQL, MySQL & blob container set up by customer. We would not be able to set it up for customer in CI. I am creating separate PR's for other type of stores.

Reply:
What's blocking us from setting this up in CI?

Reply:
Could we upload data in blob through CI?

Reply:
We can do quite a bit - what's the requirement?  Do we need any random blob? specific blob?  CI already has some blobs available...

Reply:
We could use random blob but with valid data. Can we upload data in CI blobs manually? 

Please let me know how to use it.